src/routes/[seminarFormat]/[url]/+page.server.ts

```diff
@@ -14,87 +15,100 @@ import { UPSERT_TEILNEHMER, PUBLISH_TEILNEHMER } from '$lib/graphql/mutations';
 import { verify } from 'hcaptcha';
 
 export const load: PageServerLoad = async ({ params }) => {
-	const { url } = params;
-	const body = await api(SEMINAR, { url });
-	const seminar: Seminar = body.data.seminar;
-	if (!seminar) {
-		error(404, 'Seminar nicht gefunden');
-	}
-
-	return { seminar };
+    const { url } = params;
+    const body = await api(SEMINAR, { url });
+    const seminar: Seminar = body.data.seminar;
+
+    if (!seminar) {
```

Comment:
If your customers delete the seminar from the HyGraph backend, then `seminar` will be `null` and you will hit the 404 error here.